aries-site/src/themes/aries.js

```diff
@@ -16,34 +17,599 @@ export const aries = deepMerge(hpe, {
+        },
+      },
+      3: {
+        small: {
+          size: '16px',
+          height: '16px',
+        },
+        medium: {
+          size: '20px',
+          height: '20px',
+        },
+        large: {
+          size: '24px',
+          height: '24px',
+        },
+        xlarge: {
+          size: '36px',
+          height: '36px',
+        },
+      },
+      4: {
+        small: {
+          size: '12px',
+          height: '12px',
+        },
+        medium: {
+          size: '16px',
+          height: '16px',
+        },
+        large: {
+          size: '20px',
+          height: '20px',
+        },
+        xlarge: {
+          size: '24px',
+          height: '24px',
+        },
+      },
+      5: {
+        font: {
+          weight: 600,
+        },
+        small: {
+          size: '12px',
+          height: '12px',
+        },
+        medium: {
+          size: '12px',
+          height: '12px',
+        },
+        large: {
+          size: '16px',
+          height: '16px',
+        },
+        xlarge: {
+          size: '20px',
+          height: '20px',
+        },
+      },
+      6: {
+        font: {
+          weight: 600,
+        },
+        small: {
+          size: '12px',
+          height: '12px',
+        },
+        medium: {
+          size: '12px',
+          height: '12px',
+        },
+        large: {
+          size: '12px',
+          height: '12px',
+        },
+        xlarge: {
+          size: '16px',
+          height: '16px',
+        },
+      },
+    },
+    extend: ({ level, size }) => {
+      let fontWeight = '';
+      if (level === 1 && size === 'small') {
+        fontWeight = 'font-weight: 500;';
+      } else if (level === 2 && ['large', 'xlarge'].includes(size)) {
+        fontWeight = 'font-weight: 400;';
+      } else if (level === 3 && size === 'small') {
+        fontWeight = 'font-weight: 600;';
+      } else if (level === 4 && ['small', 'medium'].includes(size)) {
+        fontWeight = 'font-weight: 600;';
+      } else if (level === 5 && size === 'xlarge') {
+        fontWeight = 'font-weight: 500;';
+      }
+      return fontWeight;
+    },
+  },
+  grommet: {
+    // items like anchor, checkbox, just inherit Grommet's base font sizes
+    // extend: `
+    //   font-size: 16px;
+    //   line-height: 20px;
+    // `,
+  },
+  pageHeader: {
+    actions: {
+      pad: { vertical: 'none' }, // pad: { vertical: 'xxsmall' }, // aligns button height with heading font-size instead of line-height
+    },
+    subtitle: {
+      size: 'xlarge', // size: 'large',
+    },
+    title: {
+      size: 'medium', // size: 'small',
+    },
+  },
+  paragraph: {
+    xsmall: {
+      size: '14px',
+      height: '16px',
+    },
+    small: {
+      size: '16px',
+      height: '18px',
+    },
+    medium: {
+      size: '18px',
+      height: '24px',
+    },
+    large: {
+      size: '22px',
+      height: '26px',
+    },
+    xlarge: {
+      size: '28px',
```

Comment:
Double check all of these values, latest proposed from Chris are slightly different: https://www.figma.com/file/wtwk7YovJMvDb3cfWCMWpu/HPE-Product-Brand-Guidelines?node-id=119%3A5816&t=M5EfR2WJoJkBWzIv-4